pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -393,6 +394,15 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		},
 		{
 			ObjectMeta: kapi.ObjectMeta{
+				Name: ImageSignerRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule("get").Groups(imageGroup).Resources("images").RuleOrDie(),
```

Comment:
I'm ok with making an image signer role sufficiently powerful to make it one stop shopping for someone trying to do it, but in that case you should add the ability to pull any image it knows about.